up-l1/zenoh.adoc

```diff
@@ -253,7 +240,8 @@ The way to transfer Response message is different from others, because Zenoh can
 ==== Message routing
 
 1. A streamer that is interested in all incoming messages for dest-authority registers a listener for
-    * `up://dest-authority////`
+    * source: `-`
```

Comment:
Can't leave the comment high up enough. I'm referring to [this line](https://github.com/eclipse-uprotocol/up-spec/blob/main/basics/uri.adoc#611-matching-on-authority).

With the separation coming between uP-L1 and uP-L2, I have a question: would the `up-client-foo-rust` libraries be responsible for / be expected to implement uP-L2? Or would they _optionally_ implement uP-L2, but allow for the posibility of plugging in the UTransport implementation they made into e.g. the RpcClient "default" implementation we'd have in `up-rust`?

Tagging @sophokles73 in for this one as well to hear his thoughts.